src/main/resources/log4j.properties

```diff
@@ -0,0 +1,16 @@
+# Root logger option
+log4j.rootLogger=INFO, stdout, file
+
+# Redirect log messages to console
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.Target=System.out
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
+
+# Redirect log messages to a log file, support file rolling.
+log4j.appender.file=org.apache.log4j.RollingFileAppender
+log4j.appender.file.File=logs/cinema.log
```

Comment:
does it work for the relative path? just curious

Reply:
![image](https://user-images.githubusercontent.com/60775248/96181528-d4f1c180-0f3c-11eb-8ee0-5d3794ec40de.png)
It's work)